src/components/AddMovie/select.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Field } from "formik";
+
+function Select(props) {
+  const { label, name, options, ...rest } = props;
+  return (
+    <div className="form-control">
+      <label htmlFor={name}>{label}</label>
+      <Field as="select" id={name} name={name} {...rest}>
+        {options.map((option) => {
```

Comment:
Mapping directly in the JSX is considered a performance heavy operation since on each render map will recreate its callback. To avoid that you can first of all extract the mapping into a separate function that returns the mapping result, and also extracting map's callback into a separate function as well - we usually do that to then add a memoization logic to the mapping function and reduce the number of map iterations. E.g. `options.map((option) => <option key={option.value} value={option.value} >...</option>)` could become `const renderOptions = options => options.map(renderOption);` and `const renderOption = option => <option key={option.value} value={option.value} >...</option>;`

Reply:
refactored the jsx accordingly and resolved.